.nancy-ignore

```diff
@@ -0,0 +1,8 @@
+# The following should be considered as temporary work-arounds
+sonatype-2020-1055
+sonatype-2019-0702
+CVE-2022-29153
+CVE-2022-29153
+sonatype-2021-1401
```

Comment:
heads up I am being told by @rafahop  to deal with them as opposed to add to list and create a ticket for future. 
What should we do in team b?